_posts/2021-06-07-changelog-148.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #148"
+date: 2021-06-07 17:00:00 +0300
+---
+
```

Comment:
Let's add a comment that it's the last release that support 2020.3 IDEs. You can find an example of similar note in some previous changelogs